lib/killbill_client/models/invoice_payment.rb

```diff
@@ -69,6 +71,22 @@ def bulk_create(external_payment = false, user = nil, reason = nil, comment = ni
                             :comment => comment,
                         }.merge(options)
       end
+
+      def chargeback_reversal(payment_id, amount, adjustments = nil, user = nil, reason = nil, comment = nil, options = {})
+        payload             = InvoicePaymentTransactionAttributes.new
+        payload.amount      = amount
+        payload.is_adjusted = !adjustments.nil?
+        payload.adjustments = adjustments
+        invoice_payment = self.class.post "#{KILLBILL_API_INVOICE_PAYMENTS_PREFIX}/#{payment_id}/chargebackReversals",
+                        to_json,
```

Comment:
`to_json ` would need to be updated to `payload.to_json` in the `post` (it's not in use today).